packages/networks/src/networks/pulsechain.ts

```diff
@@ -0,0 +1,49 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const pulsechain: NetworkConfig = {
+    chain: 'pulsechain',
+    description: 'pulsechain mainnet.',

```

Comment:
We need a better description here so users know what is specific about Pulsechain please!

Reply:
Will add something more detailed.